### ISSUE TITLE
Confirmed cases unchanged

### ISSUE THREAD

**Issue by johnbulmer** [CSSEGISandData/COVID-19#1443]
The number of confirmed cases is the same for March 23 as March 22

**Comment by yanivm** [comment]
Same bug for the number of Deaths as well as Confirmed:
- numbers for March 23 are the same as March 22.
- numbers are missing for March 23 in the states-resolution in the United States.

**Comment by jonisapp** [comment]
this is probabliy related to the fact that the headers have changed, go look inside the file from yesterday.

**Comment by brazirl** [comment]
No, they changed the file name and the formatting (including removing US data by State).  See time_series_covid19_confirmed_global.csv. This is a mess now.
